src/validation.cpp

```diff
@@ -2518,14 +2511,10 @@ bool ActivateBestChain(CValidationState &state, const CChainParams& chainparams,
 
             // throw all transactions though the signal-interface
 
-            } // MemPoolConflictRemovalTracker destroyed and conflict evictions are notified
-
             // Transactions in the connected block are notified
             for (const auto& pair : connectTrace.blocksConnected) {
                 assert(pair.second);
```

Comment:
Assert pair.first here too, I think. It'd be nice to doc that the BlockConnected() params are now always non-null.